src/hipscat/io/file_io/file_io.py

```diff
@@ -145,7 +163,8 @@ def load_parquet_to_pandas(
     Returns:
         pandas dataframe loaded from parquet
     """
-    return pd.read_parquet(file_pointer, storage_options=storage_options, **kwargs)
+    pd_storage_option = handle_pandas_storage_options(storage_options)
+    return pd.read_parquet(file_pointer, storage_options=pd_storage_option, **kwargs)
 
 
 def write_dataframe_to_csv(
```

Comment:
Should this write method also get modified to use improved storage options? I guess it's tricky to test since you're not likely using http to write files.

Reply:
I think that it's not needed on the write methods, since this is just for the `headers` of the http requests.